docs/setup/independent/high-availability.md

```diff
@@ -0,0 +1,480 @@
+        - --listen-peer-urls https://${PRIVATE_IP}:2380 \
+        - --initial-advertise-peer-urls https://${PRIVATE_IP}:2380 \
+        - --cert-file=/certs/server.pem \
+        - --key-file=/certs/server-key.pem \
+        - --client-cert-auth \
+        - --trusted-ca-file=/certs/ca.pem \
+        - --peer-cert-file=/certs/peer.pem \
+        - --peer-key-file=/certs/peer-key.pem \
+        - --peer-client-cert-auth \
+        - --peer-trusted-ca-file=/certs/ca.pem \
+        - --initial-cluster etcd0=https://<etcd0-ip-address>:2380,etcd1=https://<etcd1-ip-address>:2380,etcd1=https://<etcd2-ip-address>:2380 \
+        - --initial-cluster-token my-etcd-token \
+        - --initial-cluster-state new
+        image: gcr.io/google_containers/etcd-amd64:3.0.17
+        livenessProbe:
+        httpGet:
+            path: /health
+            port: 2379
+            scheme: HTTP
+        initialDelaySeconds: 15
+        timeoutSeconds: 15
+        name: etcd
+        env:
+        - name: PUBLIC_IP
+        valueFrom:
+            fieldRef:
+            fieldPath: status.hostIP
+        - name: PRIVATE_IP
+        valueFrom:
+            fieldRef:
+            fieldPath: status.podIP
+        - name: PEER_NAME
+        valueFrom:
+            fieldRef:
+            fieldPath: metadata.name
+        volumeMounts:
+        - mountPath: /var/lib/etcd
+        name: etcd
+        - mountPath: /certs
+        name: certs
+    hostNetwork: true
+    volumes:
+    - hostPath:
+        path: /var/lib/etcd
+        type: DirectoryOrCreate
+        name: etcd
+    - hostPath:
+        path: /etc/kubernetes/pki/etcd
+        name: certs
+    EOL
+    ```
+
+    Make sure you replace:
+    * `<podname>` with the name of the node you're running on (e.g. `etcd0`, `etcd1` or `etcd2`)
+    * `<etcd0-ip-address>`, `<etcd1-ip-address>` and `<etcd2-ip-address>` with the public IPv4s of the other machines that host etcd.
+
+{% endcapture %}
+
+{% assign tab_names = "Choose one...,systemd,Static Pods" | split: ',' | compact %}
+{% assign tab_contents = site.emptyArray | push: choose | push: systemd | push: static_pods %}
+
+{% include tabs.md %}
+
+## Set up master Load Balancer
+
+1. The next step is to create a cloud Load Balancer for your master nodes. This solution will depend on your cloud provider. Examples are:
+
+    * [AWS Elastic Load Balancer](https://aws.amazon.com/elasticloadbalancing/)
+    * [GCE Load Balancing](https://cloud.google.com/compute/docs/load-balancing/)
+    * [Azure](https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-overview)
+
+    You will need to ensure that the load balancer routes to all three master machines set up in the previous step **on port 6443**. If possible, use a smart load balancing algorithm like "least connections", and use health checks so unhealthy nodes can be removed from circulation. Most providers will provide these features.
+
+## Acquire etcd certs
```

Comment:
Similar to what suggester before for etcd: What about and grouping - indenting - all the master-reated task under an umbrella paragraph:

e.g. of the resulting Index

- Before you begin
- Setup an HA etcd cluster 
   - Create etcd CA certs
   - ...
- Set up master Load Balancer
- Set up masters
  - Installing pre-requisites on masters (the paragraph moved from above)
  - Acquire etcd certs (this paragraph)
  -Run kubeadm init on master0
  - Run kubeadm init on master1 and master2
- Install workers